Tribler/Test/Core/Upgrade/test_version_manager.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import absolute_import
+
+import os
+
+from configobj import ConfigObj
+
+from twisted.internet.defer import inlineCallbacks
+
+from Tribler.Core import version as tribler_version
+from Tribler.Core.Config.tribler_config import CONFIG_SPEC_PATH, TriblerConfig
+from Tribler.Core.Session import Session
+from Tribler.Core.Upgrade.upgrade import TriblerUpgrader
+from Tribler.Core.Upgrade.version_manager import VersionManager
+from Tribler.Core.simpledefs import STATEDIR_CHANNELS_DIR, STATEDIR_CHECKPOINT_DIR, \
+    STATEDIR_DB_DIR, STATEDIR_WALLET_DIR
+from Tribler.Test.test_as_server import AbstractServer
+
+
+class TestVersionManager(AbstractServer):
+
+    @inlineCallbacks
+    def setUp(self):
+        yield super(TestVersionManager, self).setUp()
+        self.config = TriblerConfig(ConfigObj(configspec=CONFIG_SPEC_PATH))
+        self.config.set_root_state_dir(self.getRootStateDir())
+        self.session = Session(self.config)
+
+        self.upgrader = TriblerUpgrader(self.session)
+        self.version_manager = VersionManager(self.session)
+
+    def test_read_write_version(self):
+        # If there is no version history file, no information about last version is available
+        self.assertFalse(os.path.exists(self.version_manager.version_history_path))
+        self.assertIsNone(self.version_manager.version_history['last_version'])
+        self.assertEqual(self.version_manager.version_history['history'], {})
+
+        # Saving the version
+        self.version_manager.update_version_history('7.0.0-GIT')
+        self.assertEqual(self.version_manager.version_history['last_version'], '7.0.0-GIT')
```

Comment:
Also, it is more readable to use local variables for this purpose. Otherwise, the reader is confused if `7.0.0-GIT` is expected because it is the default value, or because has been *set* to this value.